onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -480,6 +487,14 @@
         commandList->ResourceBarrier(2, barriers);
     }
 
+    std::vector<uint32_t> GetTensorDimensions(IMLOperatorTensor* tensor)
+    {
+        auto inputDimsSize = tensor->GetDimensionCount();
+        auto dims = std::vector<uint32_t>(inputDimsSize);
```

Comment:
>auto dims = std::vector<uint32_t>(inputDimsSize); [](http://example.com/codeflow?start=8&length=49)

[minor] Why not just...

`std::vector<uint32_t> dims(inputDimsSize);`

...rather than the extra `auto` and `=`?

Reply:
because auto left aligns all the variable names making it easy to read them quickly :D